### DISCUSSION TITLE
Does the audit-data folder in the vault-guardians audit contain all of the issues that the protocol has?

### DISCUSSION THREAD

**Discussion by dobrygeorgiev** [Cyfrin/security-and-auditing-full-course-s23#246] [Q&A]
In the README.md file it is stated that all of the findings in the audit-data folder are considered known. Are there more vulnerabilities within the protocol? If there are, is there a full report by the Cyfrin team that we as students can use to check after we finish reviewing the codebase?

**Comment by EngrPips** [comment]
They are known because that is the bug planted in the codebase by the author, and It is hard to confidently say that this is all the bug present in this codebase because some bugs can be really hard to catch that we don't even think they existed, and they do. So you are meant to compare your findings to what is in the `audit-data`
